tests/php/sync/test_class.jetpack-sync-terms.php

```diff
@@ -221,20 +221,12 @@ function test_returns_term_taxonomy_by_id() {
 	}
 
 	function get_terms() {
-		global $wp_version;
-		if ( version_compare( $wp_version, '4.5', '>=' ) ) {
-			return get_terms( array(
+		return get_terms(
+			array(
 				'taxonomy'   => $this->taxonomy,
 				'hide_empty' => false,
-			) );
-
-		} else {
-			return array_map( array( $this, 'upgrade_terms_to_pass_test' ), get_terms( $this->taxonomy, array(
-					'hide_empty' => false,
-				) )
-			);
-
-		}
+			)
+		);
 	}
 
 	function upgrade_terms_to_pass_test( $term ) {
```

Comment:
This method is no longer called.

Reply:
Missed that one. Done in e2aaab044dede1822d2659b1a53245afaee666ca